indexing-service/src/main/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTask.java

```diff
@@ -346,4 +347,24 @@ public LookupLoadingSpec getLookupLoadingSpec()
   {
     return LookupLoadingSpec.NONE;
   }
+
+  @Override
+  public boolean isReady(TaskActionClient taskActionClient) throws Exception
+  {
+    final boolean isUsingConcurrentLocks = Boolean.TRUE.equals(
```

Comment:
```suggestion
    final boolean useConcurrentLocks = Boolean.TRUE.equals(
```

Also, when you're adding tests, could you please add docs for this task context?